### ISSUE TITLE
Wan 2.2 T2V 14B，微调后的模型不支持多卡推理吗？推理出来全是噪声

### ISSUE THREAD

**Issue by YPatrickW** [modelscope/DiffSynth-Studio#1216]
seed同步问题，可以试试我下面的代码

**Comment by YPatrickW** [comment]
@Artiprocher 麻烦帮忙看看

**Comment by YPatrickW** [comment]
if dist.is_initialized():
    rank = dist.get_rank()
    seed_tensor = torch.tensor([random.randint(0, 100)], dtype=torch.long, device=device)
    dist.broadcast(seed_tensor, src=0)
    current_seed = seed_tensor.item()
else:
    current_seed = random.randint(0, 100)

random.seed(current_seed)
np.random.seed(current_seed)
torch.manual_seed(current_seed)
torch.cuda.manual_seed_all(current_seed)
torch.backends.cudnn.deterministic = True
